### ISSUE TITLE
Double-clicking files runs Gephi but doesn't open the files

### ISSUE THREAD

**Issue by iosonosempreio** [gephi/gephi#2594]
## Expected Behavior
A double-click on the file opens Gephi and loads the file so people can see the graph they are going to work with.

## Current Behavior
Double click opens Gephi (in case it is closed, or bring it to focus in case it is not) but does nothing else.

## Steps to Reproduce

1. Double-click on a .gephi or .gexf file
2. Observe

## Your Environment

* Version used: Gephi 0.9.7
* Operating System: MAC OS 12.1 (21C52) Apple M1 Pro

**Comment by mbastian** [comment]
Yes, this is a known issue. It has something to do with the Mac OS launcher. We essentially follow's Netbeans approach with our own approach. I opened a [ticket](https://github.com/apache/netbeans/issues/4005) on their side already. I'm no Swift expert but if you are, feel free to chime in over there.
